.github/workflows/tests.yml

```
@@ -88,6 +89,14 @@ jobs:
             python-version: '3.13'
             # https://github.com/matplotlib/matplotlib/issues/29732
             pygobject-ver: '<3.52.0'
+          - name-suffix: "(Python 3.14 pre-release)"
+            os: ubuntu-24.04
+            python-version: '3.14.0-beta.3'
```

```suggestion
            python-version: '3.14.0-dev'
```
This will pick up the latest version (currently 3.14.0-beta-3 indeed), but avoid having to change it (until the first stable version is released).